src/db.c

```diff
@@ -219,7 +219,7 @@ int dbAddRDBLoad(redisDb *db, sds key, robj *val) {
  * This function does not modify the expire time of the existing key.
  *
  * The program is aborted if the key was not already present. */
-void dbOverwrite(redisDb *db, robj *key, robj *val) {
+static void dbOverwrite(redisDb *db, robj *key, robj *val, int signal) {
```

Comment:
I'd like to suggest to rename the function and the argument
```suggestion
 *
 * The 'overwrite' flag is an indication whether this is done as part of a
 * complete replacement of they key, which can be thought of a deletion and
 * replacement. om which case we need to emit deletion signals, or just an
 * update of a value of an existing key (when false).
 *
 * The program is aborted if the key was not already present. */
static void dbSetValue(redisDb *db, robj *key, robj *val, int overwrite) {
```